holonote/annotate/annotator.py

```diff
@@ -343,8 +343,18 @@ def get_display(self, *kdims: str | hv.Dimension) -> AnnotationDisplay:
             self._displays[element_key] = self._create_annotation_element(element_key)
         return self._displays[element_key]
 
-    def __mul__(self, other: hv.Element) -> hv.Overlay:
-        return other * self.get_element(*other.kdims)
+    def __mul__(self, other: hv.Element | hv.Layout | hv.Overlay | hv.NdOverlay) -> hv.Overlay:
+        if isinstance(other, (hv.Overlay, hv.NdOverlay)):
```

Comment:
```suggestion
        if isinstance(other, (hv.Overlay, hv.NdOverlay, hv.DynamicMap)):
```

The `rasterize` function creates a dynamicmap, which also needs to be iterated through to get the kdims